Library/Formula/binwalk.rb

```diff
@@ -1,11 +1,9 @@
-require 'formula'
-
 class Binwalk < Formula
-  homepage 'http://binwalk.org/'
-  revision 1
+  homepage "http://binwalk.org/"
+  revision 2
```

Comment:
It usually isn't enough, but I'll defer to whatever the maintainers think best here. Thanks for bumping everything to SHA256 by the way! :tada:.